src/components/MainScreen/MainScreen.tsx

```diff
@@ -1,37 +1,42 @@
-import { useEffect, useState, useTransition } from "react";
+import { useEffect, useState } from "react";
 import classes from "./MainScreen.module.scss";
 import { ChuckJoke } from "../../types/ChuckJoke";
 import { jokesApi } from "../../api/jokesApi";
 import Image from "../Image/Image";
 import CategorySelect from "../CategorySelect/CategorySelect";
 import AlertSnackbar from "../AlertSnackbar/AlertSnackbar";
-import { AxiosError } from "axios";
+import { AxiosError, AxiosResponse } from "axios";
 import InputForm from "../InputForm/InputForm";
 import LanguagePicker from "../LanguagePicker/LanguagePicker";
+import SaveJokes from "../SaveJokes/SaveJokes";
 import { useTranslation } from "react-i18next";
 
 const MainScreen = () => {
   const [joke, setJoke] = useState<ChuckJoke>();
   const [errorMessage, setErrorMessage] = useState<string>();
   const [category, setCategory] = useState("");
   const [isError, setIsError] = useState(false);
-  const [customName, setCustomName] = useState<string|undefined>();
+  const [customName, setCustomName] = useState<string | undefined>();
+  const [isChuckImage, setIsChuckImage] = useState(true)
+
+  const { t } = useTranslation();
 
   useEffect(() => {
     const getJokes = async () => {
-      await fetchJokes();
+        await fetchJokes();
     };
     getJokes();
   }, []);
 
   const fetchJokes = async (name?: string) => {
-    const axiosResponse = await jokesApi.fetchRandomJokes(category,name);
-    if (axiosResponse instanceof AxiosError) {
-      setIsError(true);
-      setErrorMessage(axiosResponse.message);
-    } else {
+    try {
+      const axiosResponse = await jokesApi.fetchRandomJokes(category, name) as AxiosResponse<ChuckJoke>
```

Comment:
if you change the return type as I suggested before you won't have to add `as AxiosResponse...`